src/extensions/default/JavaScriptCodeHints/ScopeManager.js

```diff
@@ -392,6 +392,40 @@ define(function (require, exports, module) {
 
         return {promise: ternPromise};
     }
+    
+    function getRefs(fileInfo, offset) {
+        postMessage({
+            type: MessageIds.TERN_REFS,
+            fileInfo: fileInfo,
+            offset: offset
+        });
+
+        return addPendingRequest(fileInfo.name, offset, MessageIds.TERN_REFS);
+    }
+    
+    function requestFindRefs(session, document, offset) {
+        var path    = document.file.fullPath,
+            fileInfo = {type: MessageIds.TERN_FILE_INFO_TYPE_FULL,
```

Comment:
Nit: Have we used some kind of style for multiline objects? The indentation looks a bit weird, I would prefer
```js
fileInfo = {
  type: MessageIds.TERN_FILE_INFO_TYPE_FULL,
  name: path,
  offsetLines: 0,
  text: filterText(session.getJavascriptText())
};